web/static/js/main.js

```diff
@@ -439,7 +440,7 @@ $(document).ready(function () {
                 "<p class='mb-1 bold label label-success'>language:" + language + "</p>" +
                 "<span class='card-flag flag-icon flag-icon-" + flags[language] + "'></span><br>" +
                 "<p class='mb-1 bold label label-default'>scan_cmd:" + scan_cmd + "</p>" +
-                "</p>\n </a>";
+                "</p>\n </a><button style=\"background-color: #EEEEEE;padding: 8px 20px;text-decoration:none;font-weight:bold;border-radius:5px;color: #10a2ff;cursor:pointer;\"><a href=\"/results/get_json?id=" + id + "\">Get JSON</a></button>" + "<button style=\"background-color: #EEEEEE;padding: 8px 20px;text-decoration:none;font-weight:bold;border-radius:5px;color: #10a2ff;cursor:pointer;\"><a href=\"/results/get_csv?id=" + id + "\">Get CSV </a></button>";
```

Comment:
@aman566  please remove the hardcoded style from buttons here and use a css class like this:

`<button class="mb-1 bold label card-date">`